packages/nodejs/src/client.ts

```diff
@@ -49,6 +49,13 @@ export class BaseClient implements Client {
     this.storeInGlobal()
   }
 
+  /**
+   * Returns the global stored client
+   */
+  static client(): Client {
+    return global.__APPSIGNAL__
```

Comment:
How can I best access this? With `BaseClient.client()` or `Appsignal.client()`?
Would it make sense to make this a getter?

Reply:
This is currently accessed internally (within the Node.js package) with `BaseClient.client()`, and with `Appsignal.client()` from the outside.

Yes, making it a setter would be more convenient. This is just the accessor for the implementation of the missing config opt. Customers will not need to use this function once we have the new API helpers in the future.

One option is to create an accessor just for the config like `Appsignal.config` | `BaseClient.config`

Reply:
Yes, I meant "getter", sorry. 

I'll create a static getter for the client.

> To reduce one additional step of trying to access the config on the client you mean?

Yes, this may be convenient, cause internally, we'll want to access the client's config 99% of the time

Reply:
> Yes, this may be convenient, cause internally, we'll want to access the client's config 99% of the time

Ok, so we'll have two getters, one for the client, one for the config. Sounds good.